internal/testutil/keys_test.go

```diff
@@ -0,0 +1,23 @@
+package testutil_test
+
+import (
+	"testing"
+
+	"github.com/pokt-network/pocket/internal/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadLocalnetPrivateKeys(t *testing.T) {
+	keyCount := 1000
```

Comment:
This may need to be 3996 unless you are only checking validators

Reply:
Good looking out 👍

This test only exercises extracting the keys from the file, it has no notion of node types and doesn't interact with genesis.